### PR TITLE
Platform Selection Improved Code

### PR DESCRIPTION
**Pull request by harshitha-reddy-usc**

### CODE REVIEW

File: SlingingPlatformGame/Assets/Scripts/Slingshot.cs [hunk 1]
```diff
@@ -76,46 +75,46 @@ public void CreatePlatformFromIndex()
...
         }
     }
 
 
     void CreatePlatform()
     {
+        if (!remainingPlatforms.Contains(selectedPlatform))
+        {
+            if (remainingPlatforms.Count != 0) {
+                selectedPlatform = (string)remainingPlatforms[0];
```
Comment (Sai-Vaishnavi-Rachakonda) [inline SlingingPlatformGame/Assets/Scripts/Slingshot.cs:103]: won't this cause a problem when the user used up all the default platforms and still has ice ones left?

**Review by Sai-Vaishnavi-Rachakonda — COMMENTED** [review]
won't this cause a problem when the user used up all the default platforms and still has ice ones left? line 90

File: SlingingPlatformGame/Assets/Scripts/Slingshot.cs (hunk 1 above)
Comment (harshitha-reddy-usc) [reply SlingingPlatformGame/Assets/Scripts/Slingshot.cs:103]: yeah , so the change is - when default is all over, automatically ice is selected and vice versa

**Comment by harshitha-reddy-usc** [comment]
created new

**Pull request closed without merging**